generators/Program.fs

```diff
@@ -49,6 +49,9 @@ let private regenerateTestClasses options =
     
     createExercises options
     |> List.filter (shouldBeIncluded options)
+    |> (function
```

Comment:
I really like this feature! Well thought of!

I might prefer a little change to the implementation though. Perhaps you could extract the logic to a helper function, in which you can then use the `Log.Error` function to log the actual error. This is more consistent with how errors are handled elsewhere in the file.

Another option would be to do some matching:

```fsharp
let exercisesToGenerate = createExercises options |> List.filter (shouldBeIncluded options)
match exercisesToGenerate with
| [] -> Log.Error("No exercises matched given options")
| _  -> List.iter regenerateTestClass' exercisesToGenerate
```

See which ever you prefer.

Reply:
Alright, I pushed an update that's neither of those things.

I like avoiding the _"Re-generated test classes."_ message when nothing was generated, and I almost always prefer extending pipelines over declaring intermediates.

What do you think? Do you have a strong preference for one of your suggestions over this? Totally willing to change it again.